api/v1/helper.go

```diff
@@ -332,12 +330,6 @@ func NeedToUpdateSriov(ifaceSpec *Interface, ifaceStatus *InterfaceExt) bool {
 					break
 				}
 			}
-			if !ingroup && (StringInArray(vfStatus.Driver, vars.DpdkDrivers) || vfStatus.VdpaType != "") {
```

Comment:
shouldnt the fix be in in configureSriovInterface ?

that is, if a VF does not belong to any group -> bind it to default driver ?

Reply:
That can be a different way to solve the problem. I'm inclined to prefer fixing NeetToUpdateSriov because:
a. it's a little simpler. `configureSriovDevices` would have to find every VF that is not part of a group definition and rebind them
b. Suppose a user wants to keep a group of VFs for some host level networking. If he doesn't mention them in a SriovNetworkNodePolicy, the operator should not touch them.

I don't know if (b) is a valid and/or important use case, and if you know any particular reason to rebind the default driver, I'm open to that option.

Reply:
@SchSeba @ykulazhenkov thoughts ?

technically these VFs will get configured the next time they are selected.
meanwhile they will remain as is f(driver/vdpa)

i dont have an objection to current approach. more feedback is welcome.

Reply:
it looks fine for me to keep VFs configured in this case

Reply:
@adrianchiris , I added a comment to the README. Does it sound clear?
https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/710/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R252-R255